test/jdk/java/awt/regtesthelpers/PassFailJFrame.java

```diff
@@ -1097,6 +1174,35 @@ public Builder screenCapture() {
             return this;
         }
 
+        /**
+         * Adds a log area below the "Pass", "Fail" buttons.
+         * <p> The log area can be controlled by {@link #log(String)},
```

Comment:
```suggestion
         * <p>
         * The log area can be controlled by {@link #log(String)},
```
Starting on the next line makes it easier to parse when reading the source code.